app/Models/User.php

```diff
@@ -314,7 +314,10 @@ public function getActivitylogOptions(): LogOptions
     public function toSearchableArray(): array
     {
         return [
+            'banned_at' => $this->banned_at,
+            'deleted_at' => $this->Deleted,
```

Comment:
Wouldn't it make more sense to have these drive the `shouldBeSearchable` property?

Reply:
Yep, it would. Fixed in latest. To test, after pulling latest from the branch, start Horizon, and then:
```
sail artisan scout:sync-index-settings
sail artisan scout:flush "App\Models\User"
sail artisan scout:import "App\Models\User" 
```